scipy/_lib/meson.build

```diff
@@ -2,9 +2,6 @@ fs = import('fs')
 if not fs.exists('boost_math/README.md')
   error('Missing the `boost` submodule! Run `git submodule update --init` to fix this.')
 endif
-if not fs.exists('highs/README.md')
-  error('Missing the `highs` submodule! Run `git submodule update --init` to fix this.')
-endif
```

Comment:
Can you replicate that check in `scipy/optimize/_highs/meson.build` (or where appropriate)?

Reply:
Not required since `meson` will "do the right thing" with the `wrap` file if the folder is missing.